### PR TITLE
Updating .md files as a part of the CNI Website Setup

### PR DESCRIPTION
**Pull request by nate-double-u**
Updating .md files, migrating content to [the new website](https://github.com/containernetworking/cni.dev).

I think we should wait to merge this PR until the https://cni.dev site is fully up and running for continuity of documentation availability. Note, the new URLs in the README.md files point to where the site will eventually live, so we won't have to re-edit once we get the site up and the URL redirected.

Related issue: https://github.com/containernetworking/plugins/issues/548

### CODE REVIEW

**Comment by nate-double-u** [comment]
I'm not able to add reviewers yet, so @-mentioning: @squeed @bboreham @mccv1r0 @dccbw

File: pkg/ns/README.md [hunk 1]
```diff
@@ -1,41 +1,5 @@
-### Namespaces, Threads, and Go
```
Comment (squeed) [inline pkg/ns/README.md:1]: Can you keep this file? It's internal development information.

Comment (nate-double-u) [reply pkg/ns/README.md]: I've removed any files that are no longer in the website as of PR https://github.com/containernetworking/cni.dev/pull/43.
(That is, files in `pkg/` and in `vendor/`)

**Comment by nate-double-u** [comment]
I've updated the URLs in the README.md files to match the restructuring in PR https://github.com/containernetworking/cni.dev/pull/43 as well.

**Comment by nate-double-u** [comment]
I've updated the site to include the info from merge requests #546 and #551.

**Comment by nate-double-u** [comment]
The URL has been wired up, so https://www.cni.dev is live now. I think this PR is ready for merge now.

**Review by bboreham — APPROVED** [review]
Looks good to me, thanks!

**Comment by nate-double-u** [comment]
> Looks good to me, thanks!

Thanks @bboreham! Just a note, I don't have write access to this repo, so I'm not able to merge this myself.

**Pull request merged**